plugins/org.eclipse.fordiac.ide.fb.interpreter/src/org/eclipse/fordiac/ide/fb/interpreter/testappgen/CompositeTestFBGenerator.java

```diff
@@ -281,6 +281,13 @@ private void createConnToTestFB(final int index) {
 		compositeFB.getFBNetwork().getEventConnections()
 				.add(createEventConnection(runAllFB.getInterface().getEventOutputs().get(index),
```

Comment:
this is a code duplication making it hard to read. better create an EList of connections and then addAll, then also you can tell for each event connection what they are for

final EList<Connection> conns = new BasicEList<>();
Connection whatIsItAbout = ...
compositeFB.getFBNetwork().getEventConnections().addAll(conns);


also put things you need very often into helper variables, such as compositeFB.getInterfaceList().getEventInputs(), so that the code is better readable. This can be done in many many places.